cli/cmd/lib_cluster_config.go

```diff
@@ -108,6 +108,16 @@ func getClusterAccessConfig() (*clusterconfig.AccessConfig, error) {
 		}
 	}
 
+	if disallowPrompt {
```

Comment:
If there are more than one cached cluster config paths, and I append -y, I would prefer to fail with an error saying please specify cluster_name and region over defaulting to (cortex, us-west-2).